src/System.Net.Http/tests/FunctionalTests/PostScenarioTest.cs

```diff
@@ -40,79 +41,102 @@ public PostScenarioTest(ITestOutputHelper output)
             _output = output;
         }
 
-        [OuterLoop] // TODO: Issue #11345
+        [OuterLoop("Uses external servers")]
+        [Theory, MemberData(nameof(EchoServers))]
+        public async Task PostRewindableStreamContentMultipleTimes_StreamContentFullySent(Uri serverUri)
+        {
+            string requestBody = "ABC";
+
+            using (var client = new HttpClient())
```

Comment:
Do we want to document WinHttpHandler's behavior? With a comment or use `CreateHttpClient()`.

Reply:
I didn't use CreateHttpClient() because I'm not switching from the SocketsHttpHandler to the platform handlers. And that would require creating a base class and separate classes to toggle the "UseSocketsHandler" variable, etc.

The issue we're closing is just concerned with documenting SocketsHttpHandler.  It does turn out, though, that WinHttpHandler behaves the same.  It's only CurlHandler that isn't being fixed.